tests/TestHelpers/GraphHelper.php

```diff
@@ -259,6 +259,32 @@ public static function getUser(
 		);
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $adminUser
+	 * @param string $adminPassword
+	 * @param string $userName
+	 *
+	 * @return ResponseInterface
+	 */
+	public static function searchUser(
+		string $baseUrl,
+		string $xRequestId,
+		string $adminUser,
+		string $adminPassword,
+		string $userName
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'users' . "?\$search=$userName");
```

Comment:
this needs to be a pattern. we can search users using other things like `displayName`, `mail`, `id`, or `groupTypes`
```suggestion
		$url = self::getFullUrl($baseUrl, 'users' . "?\$search=$searchTerm");
```

Reply:
>we can search users using other things like `displayName`, `mail`, `id`, or `groupTypes`

I was mistaken here. this could or could not be true (I only know of displayname and mail)
The actual statement was that the search response should only contains these info